README.md

```diff
@@ -84,7 +86,8 @@ You'll probably have to hack with this application to make it work with one of t
 This project is built on [Vue.js](http://vuejs.org/) 2 for the frontend and [Flask](http://flask.pocoo.org/) (Python) for
 the backend. The [todotxtio](https://github.com/EpocDotFr/todotxtio) PyPI package is used to parse/write the Todo.txt file,
 giving/receiving data through [Ajax](https://en.wikipedia.org/wiki/Ajax_(programming)). Several storage backends
-are available so one can choose to save its Todo.txt file locally on the filesystem or on its Dropbox.
+are available so one can choose to save its Todo.txt file locally on the filesystem, on its Dropbox or in a WebDav instance
+like nextcloud..
```

Comment:
Typo: extra dot. Also maybe "NextCloud" is missing its uppercase letters.

Reply:
Addressed in latest commits.